ansible_rulebook/shared_job_monitor.py

```
@@ -0,0 +1,382 @@
+                            job_data.get("status"),
+                        )
+                        processed_count += 1
+
+            logger.debug(
+                "Batch poll chunk processed %d/%d %s",
+                processed_count,
+                len(job_ids),
+                job_type,
+            )
+
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)